src/mono/System.Private.CoreLib/src/System/Reflection/Metadata/AssemblyExtensions.cs

```diff
@@ -8,6 +8,13 @@ namespace System.Reflection.Metadata
     public static class AssemblyExtensions
     {
         [CLSCompliant(false)]
-        public static unsafe bool TryGetRawMetadata(this Assembly assembly, out byte* blob, out int length) => throw new NotImplementedException();
+        public static unsafe bool TryGetRawMetadata(this Assembly assembly, out byte* blob, out int length)
+        {
+            ArgumentNullException.ThrowIfNull(assembly);
+
+            blob = null;
+            length = 0;
+            return false;
+        }
```

Comment:
Ate we missing tests? Are tests being skipped?

Reply:
[The method is not tested at all.](https://github.com/search?q=repo%3Adotnet%2Fruntime%20TryGetRawMetadata&type=code)

Reply:
Could you add some? :)

Reply:
I added a test.